src/dali/plugin/input/csv/bitbank_supplemental.py

```diff
@@ -48,22 +51,81 @@ class InputPlugin(AbstractInputPlugin):
     def __init__(
         self,
         account_holder: str,
-        withdrawals_csv_file: str,
-        withdrawals_code: str,
+        deposits_csv_file: Optional[str] = None,
+        deposits_code: Optional[str] = None,
+        withdrawals_csv_file: Optional[str] = None,
+        withdrawals_code: Optional[str] = None,
         native_fiat: Optional[str] = None,
     ) -> None:
 
         super().__init__(account_holder=account_holder, native_fiat=native_fiat)
-        self.__withdrawals_csv_file: str = withdrawals_csv_file
+        self.__deposits_csv_file: Optional[str] = deposits_csv_file
+        self.__deposits_code: Optional[str] = deposits_code
+        self.__withdrawals_csv_file: Optional[str] = withdrawals_csv_file
         # Code of the asset being withdrawn since it is NOT included in the CSV file.
-        self.__withdrawals_code: str = withdrawals_code
+        self.__withdrawals_code: Optional[str] = withdrawals_code
         self.__logger: logging.Logger = create_logger(f"{self.__BITBANK_PLUGIN}/{self.account_holder}")
 
     def load(self) -> List[AbstractTransaction]:
-        return self.parse_withdrawals_file(self.__withdrawals_csv_file)
+        result: List[AbstractTransaction] = []
+
+        if self.__withdrawals_csv_file:
+            result.extend(self.parse_withdrawals_file(self.__withdrawals_csv_file))
+
+        if self.__deposits_csv_file:
+            result.extend(self.parse_deposits_file(self.__deposits_csv_file))
+
+        return result
+
+    def parse_deposits_file(self, file_path: str) -> List[AbstractTransaction]:
+        result: List[AbstractTransaction] = []
+        if self.__deposits_code:
+            deposits_code: str = self.__deposits_code
+        else:
+            raise Exception("Bitbank.cc supplemental plugin deposits file declared without deposits code.")
```

Comment:
Raising Exception is reserved for internal errors (or sanity errors). In the case of user errors, you can use `RP2ValueError` or `RP2TypeError`.